action.yml

```
@@ -79,6 +79,10 @@ inputs:
     required: false
     description: >-
       Path to a file containing the output of Faux Pas.
+  skip-uncommitted-files-check:
+    required: false
+    description: >-
+      skip validation on uncommited files
```

A small suggestion ;D
```suggestion
      Skip validation on uncommited files
```